frontend/catalyst/compiler.py

```diff
@@ -248,13 +248,13 @@ def run_writing_command(command: List[str], compile_options: Optional[CompileOpt
         "convert-index-to-llvm",
         "convert-catalyst-to-llvm",
         "convert-quantum-to-llvm",
+        "add-exception-handling",
```

Comment:
Is it still necessary?

Reply:
The exception handling is necessary because an exception might happen in the runtime. If we do not handle exceptions then one thread crashes while the other is keeps running. I don't remember if that means that the same error is triggered or we just land in a segfault (most likely the second one).